.scrutinizer.yml

```diff
@@ -0,0 +1,44 @@
+filter:
+    excluded_paths:
+        - tests/
+        - vendor/
+before_commands:
+    - composer install
+tools:
+    php_code_sniffer:
+        config:
+            standard: PSR2
```

Comment:
PSR-12 is an option as well: https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md
Thoughts?